features/YellowBox.feature

```diff
@@ -4,7 +4,72 @@ Feature: Managing Features with the YellowBox
   as a customer
   I want to be able approve and decline features
 
+  Background:
+    Given the store operates on a single channel in "EUR" currency
+    And I am on the yellow-box page
+    And there is no yellowbox cookie
+
   @ui
-  Scenario: Visit the shop-page
-    When I am on the yellow-box page
-    Then I should see the yellow-box overlay
\ No newline at end of file
+  Scenario: I Visit the shop-page
+    Then I should see the yellow-box overlay
+
+  @ui @javascript
+  Scenario: I open and close the overlay
+    Then I should see the yellow-box overlay
```

Comment:
Ein Scenario sollte eigentlich immer nach folgendem Prinzip aufgebaut sein:
Given ....
And ....
And ...
When ....
And ....
Then ...
And ...

Given => ist dabei die Vorbereitung
When => ist die Ausführung
Then => ist die Überprüfung der Ergebnisse

Behat behandelt die Schlüsselwörter zwar alle gleich, allerdings geht ein großer Vorteil von behat/gherkin verloren, wenn man es so durcheinander schreibt wie hier und zwar, dass man die Scenarios wie einen 'Text' lesen kann.

Und es macht in meinen Augen auch keinen Sinn, mehrmals **Given**, **When** und **Then** innerhalb eines Scenarios zu haben